### DISCUSSION TITLE
openssl daily snapshots and quitls

### DISCUSSION THREAD

**Discussion by rootnk** [quictls/openssl#74] [General]
Can this be made possible based on openssl 1.1.1X-dev and openssl-3.0.X-dev ?

**Comment by tmshort** [comment]
We don't have the bandwidth to rebase the dozens of QUIC-specific changes (and fix up conflicts) on a daily basis. Since this is a rebase (i.e. QUIC changes always on top) it would mean creating daily branches.
Using an upstream tag gives QUICTLS a known, stable point to start from.
Upstream has an unwritten goal of quarterly releases (or more frequently if needed).

**Comment by rootnk** [comment]
All right what about diff patches for 111 and 3.0 ? I can test

**Comment by tmshort** [comment]
Since this is all in git, you can create patches of the current commits using `git format-patch` or equivalent commands, using the appropriate upstream label as a base.
